docs/OCP_CI_Tutorials/Scenarios/Scenarios_Guide.md

```diff
@@ -39,47 +41,57 @@ flowchart LR
   end
 
 ```
+
 ## Provision
+
 We define provision as:
 > The initial setup of your scenarios files and the OpenShift installation
```

Comment:
```suggestion
> The initial setup of your scenario's files and the OpenShift installation
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)